includes/admin/class-wc-admin-menus.php

```diff
@@ -184,8 +184,8 @@ public function menu_order_count() {
 
 			// Add count if user has access.
 			if ( apply_filters( 'woocommerce_include_processing_order_count_in_menu', true ) && current_user_can( 'manage_woocommerce' ) ) {
-				$order_count = wc_processing_order_count();
-
+				$order_count = apply_filters( 'woocommerce_modify_processing_order_count_in_menu', wc_processing_order_count() );
+				
```

Comment:
There's some extra tabs/spaces on this line to remove.

Reply:
I reverted my changes to this file so they should be resolved now.